amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/audit/AuditDeleteHoldTests.java

```diff
@@ -109,7 +109,9 @@ public void deleteHoldEventIsAudited()
 
         STEP("Check the audit log contains the entry for the deleted hold with the hold details.");
         rmAuditService.checkAuditLogForEvent(getAdminUser(), DELETE_HOLD, rmAdmin, HOLD2,
-                Collections.singletonList(ImmutableMap.of("new", "", "previous", HOLD2, "name", "Hold Name")));
+                List.of(ImmutableMap.of("new", "", "previous", HOLD2, "name", "Hold Name"),
+                    ImmutableMap.of("new", "", "previous", "", "name", "Hold deletion reason")));
```

Comment:
I can't see where this audit event has come from. Is it a separate test? If so is there an ordering that's required for the tests now? e.g. If we run this test locally will it fail?

Reply:
The audit event has come because of the hold deletion `holdsAPI.deleteHold(rmAdmin, holdRef);`. Actually I fixed this existing test due to the fact that the information about the deletion reason is send according to the changes in this PR - in this scenario the deletion reason is empty.

Reply:
Ah - I understand now. `"Hold deletion reason"` is the name of the property, not the reason for deleting the hold.